### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by BenWhitehead**
* Update instructions on commands to perform a bump of conformance-tests
* Cleanup description of repo now that it is no longer in google-cloud-java
* Add BigTable and Storage projects to list of projects in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+
+To update the conformance tests run the following commands from the repository root:
 ```bash
-pushd conformance-tests
-git pull
-popd
-./generate-conformance-tests.sh
-git add .
+git checkout master
+git pull upstream master
```
Comment (frankyn) [inline README.md:29]: Do I need to create a branch?

`git checkout -b branch-name`

Comment (frankyn) [reply README.md:29]: Oh, https://github.com/googleapis/java-conformance-tests/blob/master/bump-conformance-tests.sh#L53 creates a new branch as well. That wasn't clear in the README.md section though.

Comment (BenWhitehead) [reply README.md:29]: I've tried to clarify this in the sentence right after this code block, can you see if that helps?

Comment (frankyn) [reply README.md:29]: Thanks, do you want to add command `git push`?

Comment (BenWhitehead) [reply README.md:29]: Done

**Pull request merged**